iac-extensions/kubernetes/src/test/resources/checks/VariableNameConventionCheck/helm/templates/template.yaml

```diff
@@ -1,11 +1,11 @@
-{{- $my_Variable := .Values.myVariableValue }} {{- /* Noncompliant {{Rename this variable "$my_Variable" to match the regular expression '^\$[a-z][a-zA-Z0-9]*$'.}} */}}
```

Comment:
Apparently, parsing Noncompliant comments doesn't work that well for Go templates and is now confusing rather than useful. We will need to follow up on this